core/src/com/mygdx/minigolf/controller/InputHandler.java

```diff
@@ -1,4 +1,62 @@
 package com.mygdx.minigolf.controller;
 
-public class InputHandler {
-}
+import com.badlogic.gdx.InputAdapter;
+import com.badlogic.gdx.math.Vector3;
+import com.mygdx.minigolf.Game;
+
+/*
+ *  Handles touch input to control movement of ball
+ */
+public class InputHandler extends InputAdapter {
+
+    private final Game game;
+
+    private boolean ballMoveOnTouchDown = false;
+
+    private final Vector3 dragStartPos = new Vector3();
+    private final Vector3 draggingPos = new Vector3(); // Use this to draw the crosshair (when ball is not moving)
+
+    public InputHandler(Game game) {
+        this.game = game;
```

Comment:
Usikker på om dette er den beste måten å få tak i game-objektet på (brukes for å få tak i cam og ballBody, som brukes i beregningene for input).

Reply:
Enig i at det burde sees nærmere på. Men tror det kan være vanskelig å komme med lure idéer før vi vet mer om hvordan gameloopen ser ut.

Reply:
En liten forbedring kunne vært å lagre ballen og kamera istedenfor game. Da slipper vi å ha getter-metoder i Game klassen og du slipper å kalle så mange get-metoder i koden her.